forms-flow-web/src/components/Modals/TaskVariableModal.js

```diff
@@ -19,15 +19,19 @@ import {
 //TBD in case of Bundle form display
 const PillList = React.memo(({ alternativeLabels, onRemove }) => {
   const { t } = useTranslation();
+  // Filter out applicationId and applicationStatus
+  const filteredVariablePills = Object.values(alternativeLabels).filter(
+    ({ key }) => key !== "applicationId" && key !== "applicationStatus"
```

Comment:
create a set for this like 

`ingnoreKeywords = new Set(["applicationId", "applicationStatus"])`
then check inside the filter like` .filter({key} => !ignoreKeywords.has(key))
`
if we need to add more keys in future we can add it in ignoreKeywords

Reply:
Eefactored.